pai-management/k8sPaiLibrary/maintaintool/docker-config-update.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation
+# All rights reserved.
+#
+# MIT License
+#
```

Comment:
Actually this is a generic json update tool, suggest you make name more generic, and move it to a more common place.

Reply:
I don't want to expose this tool to others. And in code structure, I only hope this tool could be used when deploying or manage k8s or docker.  And this tool can only handle following json format.
```
{
key: value
key: value
}
``` 

And unable to handle the next json format.

```
[ "Ford", "BMW", "Fiat", {"a": "b"} ]
```

Reply:
I think in this PR, the most concerned problem is whether we should force enable log rotation of docker for user. Now, I force enable it when deploy or add a new machine, no matter the docker is installed by pai or user. @sterowang

Reply:
Sure.

---
In reply to: [206409837](https://github.com/Microsoft/pai/pull/995#discussion_r206409837) [](ancestors = 206409837)